api-reference/10 UI Components/dxTreeList/1 Configuration/onCellPrepared.md

```diff
@@ -17,7 +17,7 @@ Information about the event that caused the function's execution.
 This column's [configuration](/api-reference/10%20UI%20Widgets/dxTreeList/1%20Configuration/columns '/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/').
 
 ##### field(e.columnIndex): Number
-The index of the column to which the cell belongs.
+The visible column index described in the following topic: [Column and Row Indexes](/Documentation/Guide/UI_Components/TreeList/Columns/Column_and_Row_Indexes/).
```

Comment:
I have a similar request for line 70 below as well. See my previous comment above.

Line 70: Allows tracking a variable and performing actions when it changes.

Reply:
Please apply your suggestion to line 70 (below) as well. Thank you.